hack/kind-up.sh

```diff
@@ -394,18 +397,21 @@ if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]] ; then
   ip_address_field="GlobalIPv6Address"
 fi
 
+host_ip="$(ip route | awk '$1 == "default" && $3 ~ /^[0-9.]+$/ { print $3; exit }')"
```

Comment:
This does not seem to work as expected. The kube-apiserver of the kind cluster is bound to localhost not the host machine.

On my machine the command returns the IP of the default route which is not localhost.
```
❯ ip route | awk '$1 == "default" && $3 ~ /^[0-9.]+$/ { print $3; exit }'
10.0.0.10
```

Something similar happens in the prow cluster.
```
root@a539cb5f-9dbc-4dbd-a9d4-aeb44fae5d95:/home/prow/go/src/github.com/gardener/gardener# ip route | awk '$1 == "default" && $3 ~ /^[0-9.]+$/ { print $3; exit }'
169.254.1.1
```

Thus, we could either try to bind the kind apiserver to `0.0.0.0` or use 127.0.0.1 instead of the host IP.